### PR TITLE
mac: MediaPlayer support and removal of Apple Remote support

### PR DESCRIPTION
**Pull request by Akemi**
this removed the old Apple Remote support, which isn't support by Apple anymore anyways. i had @ChrisK2 test with his Apple Remote on the current macOS and either it doesn't work anymore or his 3rd Part USB Infrared dongle doesn't work anymore.

Media Key support was replaced by the new MediaPlayer framework. this has the side effect that many other input devices work now too. i tested this with some Bluetooth devices and their Media Keys, they are now recognised properly. it's also possible that this re-add Apple Remote support, bt because of the lack of a working setup i can't conform this.

### CODE REVIEW

File: input/input.c [hunk 1]
```diff
@@ -196,14 +196,14 @@ const struct m_sub_options input_config = {
...
-#if HAVE_COCOA
-        OPT_FLAG("input-appleremote", use_appleremote, 0),
-#endif
 #if HAVE_SDL2_GAMEPAD
         OPT_FLAG("input-gamepad", use_gamepad, 0),
 #endif
         OPT_FLAG("window-dragging", allow_win_drag, 0),
         OPT_REPLACED("input-x11-keyboard", "input-vo-keyboard"),
+#if HAVE_COCOA
+        OPT_REMOVED("input-appleremote", "input-appleremote replaced by ..."),
```
Comment (CounterPillow) [inline input/input.c]: Is there something missing from this string?

Comment (Akemi) [reply input/input.c]: apparently yes, i was quite sure i changed that message. seems like some minor changes went missing.

**Comment by saagarjha** [comment]
Hey, just stopping by from the IINA team with a quick question: am I correct in my understanding that mpv will no longer support using the system media keys to control playback on versions of macOS prior to 10.12.2?

**Comment by Akemi** [comment]
yes, that is correct.

**Comment by saagarjha** [comment]
Yes, we disable Swift. I think we'll probably just drop support for media keys on those systems.

**Pull request merged**